CODEOWNERS

```diff
@@ -0,0 +1 @@
+/tools/sts-job-manager/  danielbankhead@google.com
```

Comment:
We do not support codeowners in the pso repo since you will not have admin access to the repo itself. Please remove this file

Reply:
Removed